stock_portfolio/stock_portfolio/views/portfolio.py

```diff
@@ -48,12 +65,12 @@ def get_portfolio_view(request):
     """Load portfolio from database and display it."""
     try:
         query = request.dbsession.query(Stock)
-        all_entries = query.all()
+        user_stocks = query.filter(Junction.account_id == request.authenticated_userid, Junction.stock_id == Stock.symbol)
```

Comment:
Just so you know, you can make use of the property that you added for the `Account` model for the relationship with `Stock` to retrieve the list of stocks.
```python
query = request.dbsession.query(Account)
user_stocks = query.filter(Account.account_id == request.authenticated_userid).one_or_none().stocks
```